### PR TITLE
added a docker compose file

### PR DESCRIPTION
**Pull request by AruneshDwivedi**

### CODE REVIEW

**Review by AruneshDwivedi — COMMENTED** [review]
.

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+version: "3"
+services:
```
Comment (dricazenck) [inline docker-compose.yaml:2]: For service you can check we have on PR implemented, could you please remove.

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    ports:
+      - "5432:5432"
+    environment:      #configure this as yor db id passwd.
+      - POSTGRES_DB=wccdb
+      - POSTGRES_USER=wccuser
+      - POSTGRES_PASSWORD=wccpassword
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+sonarqube:
```
Comment (dricazenck) [inline docker-compose.yaml:27]: Could you update README to use docker compose, please and also add a description in your PR.

**Comment by dricazenck** [comment]
Thanks for the contribution @AruneshDwivedi

File: docker-compose.yaml (hunk 2 above)
Comment (AruneshDwivedi) [reply docker-compose.yaml:27]: ok.

**Comment by dricazenck** [comment]
Hi @AruneshDwivedi could please fix the comments so we can merge your PR.

**Comment by dricazenck** [comment]
Unfortunately I don't any update, I am going to close the pr.

**Pull request closed without merging**